pkg/apis/sriovnetwork/v1/sriovnetworknodepolicy_types.go

```diff
@@ -21,6 +21,9 @@ type SriovNetworkNodePolicySpec struct {
 	NicSelector  SriovNetworkNicSelector `json:"nicSelector"`
 	DeviceType   string                  `json:"deviceType,omitempty"`
 	IsRdma       bool                    `json:"isRdma,omitempty"`
+	TotalVfs     int                     `json:"totalVfs,omitempty"`
+	LinkType     string                  `json:"linkType,omitempty"` 
```

Comment:
This `LinkType` is only supported by MLX nics, right? I am not quite sure if we shall add it. As you know, we are not going to support infiniband in this release, so I don't think we should expose this mode select option, at least for now. @zshi-redhat what do you think?